src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    public static class Task {
+        protected String description;
+        protected boolean isDone;
+        protected char taskType;
+
+        public Task(String description) {
+            this.description =  description;
+            this.isDone = false;
+        }
+
+        public String getStatusIcon() {
+            return (isDone ? "X" : " ");
+        }
+        public void markAsDone() {
+            this.isDone = true;
+        }
+
+    }
+
+    static String exitTrigger = "bye";
+    static String listTrigger = "list";
+    static String doneTrigger = "done";
+    static Task[] commands = new Task[100];
+
+    public static void greet() {
+        String logo = "       ___                             | '  \\\n" +
+                "   ___  \\ /  ___         ,'\\_           | .-. \\        /|\n" +
+                "   \\ /  | |,'__ \\  ,'\\_  |   \\          | | | |      ,' |_   /|\n" +
+                " _ | |  | |\\/  \\ \\ |   \\ | |\\_|    _    | |_| |   _ '-. .-',' |_   _\n" +
+                "// | |  | |____| | | |\\_|| |__    //    |     | ,'_`. | | '-. .-',' `. ,'\\_\n" +
+                "\\\\_| |_,' .-, _  | | |   | |\\ \\  //    .| |\\_/ | / \\ || |   | | / |\\  \\|   \\\n" +
+                " `-. .-'| |/ / | | | |   | | \\ \\//     |  |    | | | || |   | | | |_\\ || |\\_|\n" +
+                "   | |  | || \\_| | | |   /_\\  \\ /      | |`    | | | || |   | | | .---'| |\n" +
+                "   | |  | |\\___,_\\ /_\\ _      //       | |     | \\_/ || |   | | | |  /\\| |\n" +
+                "   /_\\  | |           //_____//       .||`  _   `._,' | |   | | \\ `-' /| |\n" +
+                "        /_\\           `------'        \\ |               `.\\  | |  `._,' /_\\\n" +
+                "                                       \\|                    `.\\\n" +
+                "  S. Lu - I solemnly swear that I am up to no good.\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\n"
+                + "What can I do for you?\n");
+    }
+
+    public static void exit() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    //index is the position in string, index+1 is the bullet number
+    public static void add(String command, int index) {
+        commands[index] = new Task(command);
+        commands[index].taskType = getTask(command,index);
+        System.out.println("Got it. I've added this task: \n" +
+                "\t[" + commands[index].taskType + "]" + "[ ] " +
+                commands[index].description + "\n" +
+                "Now you have " + (index+1) + (index == 0?" task":" tasks") + " in the list."
```

Comment:
Few fundamental issues with this piece of code:
1. Having such complex computational logic in your print statements makes the code extremely long and hard to understand. The sole purpose of a print statement is to print something and nothing else. Therefore, its better if you separate these nested functions to different lines.
2. Also avoid using ternary operators as they can make your code unnecessarily hard to understand. It would be better to change the logic to use if ... else if statements instead and then conditionally print
3. Please maintain spacing between the operators and the operands. Your style of using spacing in code seems to be quite inconsistent